javase/src/main/java/com/google/zxing/client/j2se/CommandLineEncoder.java

```diff
@@ -46,9 +48,13 @@ public static void main(String[] args) throws Exception {
     if (EncoderConfig.DEFAULT_OUTPUT_FILE_BASE.equals(outFileString)) {
       outFileString += '.' + config.imageFormat.toLowerCase(Locale.ENGLISH);
     }
-    
+ 
+    HashMap<EncodeHintType,String> hints = new HashMap<EncodeHintType,String>();
```

Comment:
Nits: use `Map` as the variable type, and omit the generic types after `HashMap`. You're also missing spaces before arguments as in ",hints)" instead of ", hints)"